src/helix.idea.vim

```diff
@@ -0,0 +1,198 @@
+"""""""""""""""""""""""""
+" IdeaVim support       "
+"""""""""""""""""""""""""
+set nohlsearch                    " no highlight search results
+set number                        " line numbers
+set showmode                      " show current mode
+set gdefault                      " substitute all occurrences in line per default
+set history=10000                 " keep x lines of command line history
+set scrolloff=5 sidescrolloff=10  " keep some lines before and after the cursor visible
+set visualbell                    " no beep
+set smartcase                     " no ignore case when pattern is uppercase
+set incsearch                     " show search results while typing
+set wrapscan                      " searches wrap around the end of the file
+
+set ideajoin
+set idearefactormode=keep
+
+" Plugins
+set commentary
+set highlightedyank
+set argtextobj
+set matchit
+set multiple-cursors
+
+nnoremap , <Esc><Esc>
+xnoremap , <Esc><Esc>
+vnoremap , <Esc><Esc>
+
+xnoremap <Esc>o o
+nnoremap o o
+
+nnoremap d vd
+xnoremap d dv
+
+" Movement
+unmap <Left>
```

Comment:
Helix does support navigation with arrows

Reply:
It does but for some reason the mappings prevented selection with arrows. There's a few IdeaVim specific `set` options that seemed related (I have them on the default setting).

My intention with using `unmap` was to clear the remaps in `helix.vim`, but based on your comment, I assume this _should_ disable arrow completely? They do still work.

Would the correct way to do this be to do this?

```vim
map <Up> <Up>
map <Down> <Down>
map <Left> <Left>
map <Right> <Right>
```

Reply:
Oh, I guess unmapping them then results in getting the native Idea behavior then. That's also fine of course, I was just confused to see this, because I assumed that the IdeaVim default bindings for arrows would just continue to work. They do for Vim and VsVim at least.

Reply:
Yep it feels confusing indeed. Yesterday I tried various ways with the `set` options but found no combination that would make the arrows behave just like hjkl when in visual mode. 

Perhaps there is something I didn't yet try, but even with this unmapping, I lose the selection if I switch from hjkl to arrows or vice versa.

Reply:
Hm, you're right that extend mode doesn't work for arrow keys, because I neglected to map them in helix.vim